tests/structure.test.ts

```diff
@@ -16,8 +16,8 @@ describe('Internal tree structure', () => {
   });
 
   it('Creates a tree', () => {
-    const data: number[] = [0];
-    for (let i = 1; i < 11; ++i) {
```

Comment:
Is there any impact of this change function wise, or just style preference?

Reply:
it starts from 0-length arrays now, which means we support empty MVectors